packages/twenty-server/src/core/user/user.entity.ts

```diff
@@ -34,6 +34,10 @@ export class User {
   @Column()
   email: string;
 
+  @Field({ nullable: true })
```

Comment:
so I think we should not have this in this PR. It should be added in the other one :)